pkg/endpoint/bpf.go

```diff
@@ -246,53 +247,124 @@ func hashHeaderfile(hashWriter hash.Hash, filepath string) (hash.Hash, error) {
 	return hashWriter, nil
 }
 
+// AckedRedirect is a map value type that contains minimal state that
```

Comment:
Cf. my comments on `daemon/daemon.go`, I would revert all your changes to `pkg/endpoint/bpf.go`. They are redundant with and less general than my implementation in #5451.

Reply:
right, reverting these changes.